### PR TITLE
[S] Repaths blindfolds to prevent regular sunglasses creation exploit

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
fix: You can no longer turn blindfolds into regular sunglasses.
/:cl:

Blindfolds don't actually need to inherit anything from sunglasses, from what
I can see, unless there's some bad type checking somewhere in the code that
I'm not aware of.

Exploit is: cloth makes blindfold + medHUD -> medHUD sunglasses -> remove medHUD, regular sunglasses.

### CODE REVIEW

**Comment by lordpidey** [comment]
Check to make sure blindfolds still protect you from flashes, both normal, and rev.

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -253,42 +253,42 @@
 	weldingvisortoggle(user)
 
 
-/obj/item/clothing/glasses/sunglasses/blindfold
+/obj/item/clothing/glasses/blindfold
```
Comment (ShizCalev) [inline code/modules/clothing/glasses/_glasses.dm]: suglasses has
```
darkness_view = 1
dog_fashion = /datum/dog_fashion/head
```

**Comment by subject217** [comment]
we technically have a way to blacklist subtypes from crafting now, is it worth doing that over repathing?

**Comment by ExcessiveUseOfCobblestone** [comment]
absolutely

**Comment by coiax** [comment]
Blindfolds being children of sunglasses makes no sense in the first place

**Comment by ExcessiveUseOfCobblestone** [comment]
you don't type base on reality, you type base on functionality.

That said, if no functionality is lost and you're not putting sunglasses code into the new type to compensate for the change then more power to u.

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -328,7 +328,7 @@
 
 /datum/quirk/blindness/on_spawn()
 	var/mob/living/carbon/human/H = quirk_holder
-	var/obj/item/clothing/glasses/sunglasses/blindfold/white/glasses = new(get_turf(H))
+	var/obj/item/clothing/glasses/blindfold/white/glasses = new(get_turf(H))
```
Comment (Jalleo) [inline code/datums/traits/negative.dm]: This var might get confusong for some people. Its just a readability complaint.

**Review by Jalleo — APPROVED** [review]
Looks good. Ibet thos was created on purpose to beat revs somehow.

**Review by SpaceManiac — DISMISSED** [review]

**Comment by SpaceManiac** [comment]
merge conflict

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**